### PR TITLE
Create authenticator API and custom app authenticator

### PR DESCRIPTION
**Pull request by lalithapidaparthi-okta**
## Description:
- **What's changed?** 

1. Added dev docs for create authenticator API at https://developer.okta.com/docs/reference/api/authenticators-admin/
2. Document custom app authenticator as one of the supported authenticators.
3. Added create authenticator API to the postman collection.

- **Is this PR related to a Monolith release?** Monthly Release 2022.08.0 CF

- **Netlify Link:**
https://62d5e232195e54009f39a695--reverent-murdock-829d24.netlify.app/docs/reference/api/authenticators-admin/

### Resolves:

* OKTA-478768, OKTA-505841

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/api/authenticators-admin/index.md [hunk 1]
```diff
@@ -23,21 +23,123 @@ Okta Identity Engine currently supports Authenticators for the following factors
...
+    "name": "Duo Security",
+    "provider": {
+        "type": "DUO",
+        "configuration": {
+            "userNameTemplate": {
+                "template": "oktaId"
+            },
+            "integrationKey": "testIntegrationKey",
+            "secretKey": "testSecretKey",
+            "host": "{Duo Org Hostname}"
```
Comment (antonioruberto-okta) [inline packages/@okta/vuepress-site/docs/reference/api/authenticators-admin/index.md:86]: We should call this DUO Api hostname to match what you see in DUO's website when you configure the Okta app

image

File: packages/@okta/vuepress-site/docs/reference/api/authenticators-admin/index.md [hunk 2]
```diff
@@ -838,3 +950,113 @@ The Authenticator object defines the following properties:
...
+```
+{
+    "type": "app",
+    "id": "aut5gnvcjUHIWb25J0g4",
+    "key": "duo",
+    "status": "ACTIVE",
+    "name": "Duo Security",
+    "created": "2022-07-15T21:14:02.000Z",
+    "lastUpdated": "2022-07-15T21:14:02.000Z",
+    "settings": {},
```
Comment (antonioruberto-okta) [inline packages/@okta/vuepress-site/docs/reference/api/authenticators-admin/index.md:965]: Not sure why there is an empty settings object - normally should be omitted ...

File: packages/@okta/vuepress-site/docs/reference/api/authenticators-admin/index.md (hunk 1 above)
Comment (lalithapidaparthi-okta) [reply packages/@okta/vuepress-site/docs/reference/api/authenticators-admin/index.md:86]: I thought about that, but we call it `The Duo Security org hostname` [here in the dev docs](https://developer.okta.com/docs/reference/api/authenticators-admin/#authenticator-properties). So I changed it to be that. If needed, I can change both to API Hostname. We call it API Hostname in the product help.

File: packages/@okta/vuepress-site/docs/reference/api/authenticators-admin/index.md (hunk 2 above)
Comment (lalithapidaparthi-okta) [reply packages/@okta/vuepress-site/docs/reference/api/authenticators-admin/index.md:965]: Will check. This was returned in the response, so included it here.

**Pull request merged**